.gitignore

```diff
@@ -105,3 +105,5 @@ venv.bak/
 
 # databases
 *.sqlite3
+
+.idea/
```

Comment:
I found recently that you can add a global `.gitignore` config to your machine like this:

```bash
touch ~/.gitignore
git config --global core.excludesfile ~/.gitignore
```

Helps prevent conflicts with developer dotfiles/dotdirs like these :) No problem including this though.